File: Coding Exercises/Python/Day61/is_palindrome.py
def isPalindrome(number):
    if number < 0:
        return False

    originalNumber = number
    reversedNumber = 0
    while number > 0:
        digit = number % 10
        reversedNumber = reversedNumber * 10 + digit
        number //= 10

    return originalNumber == reversedNumber

File: Coding Exercises/TypeScript/Day61/isPalindrome.ts
function isPalindrome(number: number): boolean {
  if (number < 0) {
      return false;
  }

  let originalNumber: number = number;
  let reversedNumber: number = 0;

  while (number > 0) {
      let digit: number = number % 10;
      reversedNumber = reversedNumber * 10 + digit;
      number = Math.floor(number / 10);
  }

  return originalNumber === reversedNumber;
}
